src/Sylius/Bundle/ReviewBundle/.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: php
+
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+
+before_script: composer install --prefer-source --no-interaction
+
+script: bin/phpspec run -fpretty --verbose
+
+notifications:
```

Comment:
Please adjust whole .travis.yml file according to the rest of bundles/components.